WordPress/twentytwentyfour
# Add portfolio template type

**Description**

A step towards proving out how we can achieve alternate templates for https://github.com/WordPress/twentytwentyfour/issues/100. The flow I used in the screen recording is for new templates, but this setup will be leveraged for template replacing as well (pick the portfolio view for your existing front-page template for example). Probably also index template (but we need the replace flow to experience that). 

> We may need to have a portfolio page pattern with header and footer parts, so it's a proper full replacement — whereas currently the portfolio page pattern is just the content (like other page patterns). We may need both (which shouldn't be a heavy lift, as we're building with pattern blocks anyhow).

**Screenshots**

https://github.com/WordPress/twentytwentyfour/assets/1813435/1f0bae08-1c7c-49dc-a5ca-8946d0b25662


**Testing Instructions**


1. Activate the theme. 
2. Remove a custom front-page template if you have one. 
3. Add a new front-page template. 
4. See the portfolio home template as a choice for the template.
